src/Core/src/Configuration.php

```diff
@@ -56,6 +59,10 @@ final class Configuration
             self::OPTION_WEB_IDENTITY_TOKEN_FILE => 'AWS_WEB_IDENTITY_TOKEN_FILE',
             self::OPTION_ROLE_SESSION_NAME => 'AWS_ROLE_SESSION_NAME',
         ],
+        [
+            self::OPTION_ECS_AUTH_ENDPOINT => 'AWS_CONTAINER_CREDENTIALS_RELATIVE_URI',
```

Comment:
Please use a name close to the env variable `OPTION_CONTAINER_CREDENTIALS_RELATIVE_URI`
moreover, it's not an endpoint (no scheme, nor host) but a path